blog/2024/10-26-jellyfin-release-10.10.0/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Jellyfin 10.10.0
+description: We are pleased to announce the latest stable release of Jellyfin, version 10.10.0!
+authors: joshuaboniface
+slug: jellyfin-release-10.10.0
+tags: [release]
+---
+
+We are pleased to announce the latest stable release of Jellyfin, version 10.10.0!
+
+This major release brings several new features, improvements, and bugfixes to improve your Jellyfin experience. With our faster release cadence between 10.9.0 and 10.10.0, this release should be far less daunting, so please read on for a quick peek at what's new!
+
+You may upgrade your Jellyfin instances at any time now. For those who were running Unstable builds for testing, we thank you immensely, and you may now switch back to the Stable repository and forcibly reinstall/repull the latest version. As always, **ensure you back up your Jellyfin data and configuration directories before upgrading**. With a major release, it's possible you will hit a bug and want to revert, and to do so, you will need to restore from a backup.
+
+Happy watching!
+
+\- Joshua
+
+{/* truncate */}
+
+## Breaking Changes
+
+* There are no known client support issues in the transition from 10.9.0 to 10.10.0 - all your clients should continue to work as-is without any issues or any forced upgrades, though this may change in the future.
+
+* We have deprecated Raspberry Pi V4L2 hardware transcoding support, and are looking to deprecate Raspberry Pi (i.e. ARMv7) in general for releases past 10.10.0. Raspberry Pi has always been a tricky platform for us, as it's popular but extremely poor in performance, and this has caused a lot of people a lot of headaches. With the RPi5 entirely removing a hardware acceleration engine and the aging of the RPi4, we are taking this action now to ensure it's widely known. This also means that 10.10.x will likely be the last major release of Jellyfin to support 32-bit ARM (`armhf` in Debuntu land), which we will likely drop for 10.11.0. If you are running Jellyfin on old ARM hardware, now is the time to start upgrading. See [PR #1148](https://github.com/jellyfin/jellyfin.org/pull/1148).
+
+* We now use the system temporary storage engine (e.g. `/tmp` on Linux) for storing temporary files, to allow us to leverage temporary ramdisks and the like and avoid cluttering potentially slow storage. This may cause issues if you specifically depended on the previous behaviour. See [PR #12226](https://github.com/jellyfin/jellyfin/pull/12226).
+
```

Comment:
Maybe state that this is not the transcoding cache and is mostly used for intermediate files for scheduled tasks which is small and will be cleaned up after the task.

Reply:
I feel like that's pretty clear, "transcoding cache" is not "temporary files", but I can tweak the wording slightly.